platform/android/MapLibreAndroidTestApp/src/main/java/org/maplibre/android/testapp/activity/turf/WithinExpressionActivity.kt

```diff
@@ -126,17 +107,47 @@ class WithinExpressionActivity : AppCompatActivity() {
                     "poi-label"
                 )
         )
+
+        // Un-comment to repeatedly shuffle the points of the bound as a test of fills and earcut
```

Comment:
Can we turn this into an automated regression test somehow?

Reply:
Yeah, it probably makes more sense as a unit test.  We never turned up any actual problem with earcut, I think the reports were a red herring.  The handling of degenerate polygons is likely part of the test suite for that project anyway, so this may be fully superfluous, anyway.